Jenkinsfile

```diff
@@ -36,7 +36,7 @@ node('cloud') {
                             sh 'cp images/ubuntu-x86_64-raw.img factory/images/liquid/disk.img'
                             sh 'factory/factory run --share setup:/mnt/setup --share factory/images/liquid:/mnt/liquid /mnt/setup/bin/with-image-chroot /mnt/liquid/disk.img bash /opt/setup/ci/prepare-image-for-testing'
                             sh 'echo \'{"login": {"username": "liquid-admin", "password": "liquid"}}\' > factory/images/liquid/config.json'
-                            sh 'factory/factory run --image liquid --smp 2 --memory 2048  --share setup:/mnt/setup PYTHONUNBUFFERED=yeah /mnt/setup/bin/run_first_boot_tests.py'
+                            sh 'factory/factory run --image liquid --smp 2 --memory 2048  --share setup:/opt/setup PYTHONUNBUFFERED=yeah /opt/setup/bin/run_first_boot_tests.py'
```

Comment:
Why do we even mount `/opt/setup`? It's already in the image, and should be the same version.

Reply:
so we get the merged branch that jenkins provides, so we test against what happens after we merge the PR